src/view/document.js

```diff
@@ -256,6 +256,17 @@ export default class Document {
 	}
 
 	/**
+	 * Focuses document. It will focus {@link engine.view.EditableElement EditableElement} that is currently having
+	 * selection inside.
+	 *
+	 * @fires engine.view.Document#focus
+	 */
+	focus() {
+		this.fire( 'focus' );
```

Comment:
We may simplify the `focus` event to make it possible to fire event with the same properties. On the other hand them we will lose cohesion between various view event based on the native DOM event. But, again, it would useful feature to be able to fire these view events manually, so the whole view events API should have native DOM events as optionally and provide a helper to fire a view event.